docs/src/index.md

```diff
@@ -546,8 +546,10 @@ when reading or writing.
 Below we include a limited number of API references. Note not all of these are public interfaces, thus proceed with caution.
 
 ```@index
+Pages = ["index.md"]
```

Comment:
Unintentional ?

Reply:
Intentional — I think it's required to ignore all the documented functions in `api_bindings.md` (but maybe I'm misreading Documenter.jl's docs).

Reply:
No you're right here.